bindings/SofaTypes/src/SofaPython3/SofaTypes/Binding_Mat.cpp

```diff
@@ -44,11 +44,11 @@ static void bindSquaredMat(py::class_<Mat<R, C, double>> & /*p*/) {}
 
 template <sofa::Size S> static void bindSquaredMat(py::class_<Mat<S, S, double>> &p) {
     typedef Mat<S, S, double> MatClass;
-    p.def("identity", &MatClass::identity);
-    p.def("Identity", &MatClass::Identity);
+    p.def("identity", [](MatClass &mat){ return mat.identity(); });
+    p.def("Identity", []() { return MatClass::Identity(); });
```

Comment:
This line catch my eye because I think def_static should be used to define a binding for a static method.